lib/json_matchers/matcher.rb

```diff
@@ -24,6 +24,8 @@ def matches?(response)
       false
     rescue JSON::ParserError
       raise InvalidSchemaError
+    rescue MultiJson::ParseError
+      raise InvalidSchemaError
```

Comment:
Thanks for the patch!

What do you think about combining the errors on a single line:

```rb
rescue JSON::ParserError, MultiJson::ParseError
  raise InvalidSchemaError
end
```